lib/generator/template_values.rb

```diff
@@ -15,8 +17,12 @@ def get_binding
       binding
     end
 
+    def exercise_name
+      exercise.name
+    end
+
     def exercise_name_camel
-      exercise_name.split('_').map(&:capitalize).join
+      exercise.name.camel_case
```

Comment:
`slug` is the canonical value for problem identity.
This should be `exercise.slug.camel_case`
But either way it's a Demeter violation, so should be just `exercise.classname`

`exercise_name_camel` is a slightly misleading method name here given it's usage is to determine the test class name used in the template.